eth/downloader/downloader.go

```diff
@@ -1337,9 +1334,7 @@ func (d *Downloader) fetchNodeData() error {
 					return
 				}
 				// Processing succeeded, notify state fetcher and processor of continuation
```

Comment:
Please drop "and processor" from the comment, that part is not in sync with the code any more.